builtin/logical/pki/backend.go

```diff
@@ -0,0 +1,67 @@
+package pki
+
+import (
+	"strings"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+// Factory creates a new backend implementing the logical.Backend interface
+func Factory(map[string]string) (logical.Backend, error) {
+	return Backend(), nil
+}
+
+// Backend returns a new Backend framework struct
+func Backend() *framework.Backend {
+	var b backend
+	b.Backend = &framework.Backend{
+		Help: strings.TrimSpace(backendHelp),
+
+		PathsSpecial: &logical.Paths{
+			Root: []string{
+				"config/*",
+				"revoked/*",
```

Comment:
After discussion with our security guy we couldn't figure out a reason why non-administrators would ever need to fetch a revoked cert. However, we thought it may be useful for an administrator to fetch it, in case of some kind of forensic analysis of a break-in...revoke ASAP, but maybe later you realize you wanted a copy of it (but already rotated it on the client). So we couldn't figure out a reason not to have it, but also couldn't figure out a reason non-root users would need it.

Note that when the CRL is rotated, revoked certs will be removed if they are expired. So far other certs are left alone, for archival reasons. I'm open to changing this behavior if there are strong feelings about it...either keeping all certs, or removing all certs upon expiration.